.github/scripts/openapi-specs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+if ! base_url="http://$(docker compose ${2:-} port "${1}" 8080)"; then
```

Comment:
Please use var arg expansion here, if possible:
```suggestion
if ! base_url="http://$(docker compose ${@:2} port "${1}" 8080)"; then
```